docs/releases/6.0.txt

```diff
@@ -269,6 +269,12 @@ Miscellaneous
 Features deprecated in 6.0
 ==========================
 
+Email
+-----
+
+* Setting :setting:`ADMINS` or :setting:`MANAGERS` to a list of (name, address)
+  tuples is deprecated. Change to a list of email address strings.
```

Comment:
Might be worth mentioning here that the `name` has been unused for a while, so people don't feel the need to crawl into code to figure out what they should do with `name`s